src/main/java/emu/grasscutter/game/quest/QuestManager.java

```diff
@@ -237,64 +182,36 @@ public void triggerEvent(QuestTrigger condType, String paramStr, int... params)
             .toList();
         switch (condType) {
             //accept Conds
-            case QUEST_COND_STATE_EQUAL:
-            case QUEST_COND_STATE_NOT_EQUAL:
-            case QUEST_COND_COMPLETE_TALK:
-            case QUEST_COND_LUA_NOTIFY:
-            case QUEST_COND_QUEST_VAR_EQUAL:
-            case QUEST_COND_QUEST_VAR_GREATER:
-            case QUEST_COND_QUEST_VAR_LESS:
-            case QUEST_COND_PLAYER_LEVEL_EQUAL_GREATER:
-            case QUEST_COND_QUEST_GLOBAL_VAR_EQUAL:
-            case QUEST_COND_QUEST_GLOBAL_VAR_GREATER:
-            case QUEST_COND_QUEST_GLOBAL_VAR_LESS:
+            case QUEST_COND_STATE_EQUAL, QUEST_COND_STATE_NOT_EQUAL, QUEST_COND_COMPLETE_TALK, QUEST_COND_LUA_NOTIFY, QUEST_COND_QUEST_VAR_EQUAL, QUEST_COND_QUEST_VAR_GREATER, QUEST_COND_QUEST_VAR_LESS, QUEST_COND_PLAYER_LEVEL_EQUAL_GREATER, QUEST_COND_QUEST_GLOBAL_VAR_EQUAL, QUEST_COND_QUEST_GLOBAL_VAR_GREATER, QUEST_COND_QUEST_GLOBAL_VAR_LESS -> {
                 for (GameMainQuest mainquest : checkMainQuests) {
                     mainquest.tryAcceptSubQuests(condType, paramStr, params);
                 }
-                break;
+            }
 
             //fail Conds
-            case QUEST_CONTENT_NOT_FINISH_PLOT:
+            case QUEST_CONTENT_NOT_FINISH_PLOT -> {
                 for (GameMainQuest mainquest : checkMainQuests) {
                     mainquest.tryFailSubQuests(condType, paramStr, params);
                 }
-                break;
+            }
             //finish Conds
-            case QUEST_CONTENT_COMPLETE_TALK:
-            case QUEST_CONTENT_FINISH_PLOT:
-            case QUEST_CONTENT_COMPLETE_ANY_TALK:
-            case QUEST_CONTENT_LUA_NOTIFY:
-            case QUEST_CONTENT_QUEST_VAR_EQUAL:
-            case QUEST_CONTENT_QUEST_VAR_GREATER:
-            case QUEST_CONTENT_QUEST_VAR_LESS:
-            case QUEST_CONTENT_ENTER_DUNGEON:
-            case QUEST_CONTENT_ENTER_ROOM:
-            case QUEST_CONTENT_INTERACT_GADGET:
-            case QUEST_CONTENT_TRIGGER_FIRE:
-            case QUEST_CONTENT_UNLOCK_TRANS_POINT:
-            case QUEST_CONTENT_SKILL:
+            case QUEST_CONTENT_COMPLETE_TALK, QUEST_CONTENT_FINISH_PLOT, QUEST_CONTENT_COMPLETE_ANY_TALK, QUEST_CONTENT_LUA_NOTIFY, QUEST_CONTENT_QUEST_VAR_EQUAL, QUEST_CONTENT_QUEST_VAR_GREATER, QUEST_CONTENT_QUEST_VAR_LESS, QUEST_CONTENT_ENTER_DUNGEON, QUEST_CONTENT_ENTER_ROOM, QUEST_CONTENT_INTERACT_GADGET, QUEST_CONTENT_TRIGGER_FIRE, QUEST_CONTENT_UNLOCK_TRANS_POINT, QUEST_CONTENT_SKILL -> {
```

Comment:
why even keep these switch cases? Just change it to always trigger fail and finish.
if you want to still log unhandled trigger, do this in the default handler that always resturns false

Reply:
will consider when merging dev back into unstable